dnf-behave-tests/features/install-installed.feature

```diff
@@ -46,6 +52,31 @@ Scenario: Install installed RPM when upgrade is available with best=True (defaul
         | upgrade       | glibc-common-0:2.28-26.fc29.x86_64        |
 
 
+@bz1670776 @bz1671683
+Scenario: Install installed RPM when upgrade is available with best=False (in dnf.conf)
+  Given I use the repository "dnf-ci-fedora"
+    And I do not set config file
+    And I create file "/etc/dnf/dnf.conf" with
+    """
+    [main]
+    best=True
```

Comment:
best=False !
or alternatively:
```
gpgcheck=1
installonly_limit=3
clean_requirements_on_remove=True
best=False
```

Reply:
No, this tests distribution with other defaults (like RHEL).